editor/modes/visual-editor/block-list.js

```diff
@@ -106,9 +101,8 @@ export default connect(
 		selectionEnd: getBlockSelectionEnd( state ),
 	} ),
 	( dispatch ) => ( {
-		clearSelectedBlock: () => dispatch( { type: 'CLEAR_SELECTED_BLOCK' } ),
 		onMultiSelect( { start, end } ) {
 			dispatch( { type: 'MULTI_SELECT', start, end } );
 		},
 	} )
-)( clickOutside( VisualEditorBlockList ) );
```

Comment:
I'd think of the multi-select header as having `tabIndex="-1"` and being some weird kind of modal. It would contain the focus, and if you click outside of it, or press escape, the selection is cleared, just like a modal would be closed. As long as the selection is there, backspace always deletes the selection content.